src/App.tsx

```diff
@@ -1,24 +1,82 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Section } from './components/Section';
+import { Footer } from './components/Footer';
+import * as postService from './api/todos';
+import { Todo } from './types/Todo';
+import { TasksFilter } from './types/tasksFilter';
+import { Errors } from './components/Errors';
 
-const USER_ID = 0;
+const USER_ID = 12147;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tasksFilter, setTasksFilter] = useState<TasksFilter>(TasksFilter.all);
+  const [errorId, setErrorId] = useState(0);
```

Comment:
Use magic numbers is unclear - my assumption use error message instead of magic id